app/controllers/static_contents_controller.rb

```diff
@@ -6,6 +6,18 @@ class StaticContentsController < ApplicationController
   before_action :authenticate_user!
   before_action :authenticate_admin
 
+  include SortableController
+
+  def params_for_link(link_type, value=nil)
+    if link_type == :type
+      p = super()
```

Comment:
bitte auch hier längere Variablennamen. (siehe unten)

In beiden Fellen dazu: 
`p` ist bereits vorbelegt in Ruby (https://apidock.com/ruby/Kernel/p) zur Ausgabe von Inhalten in STDOut

Reply:
da wir hier keine klassische Vererbung nutzen (StaticContentsController erbt von ApplicationController und nicht von SortableController) finde ich `super()` an der stelle schwierig zu verstehen woher es kommt (aus SortableController)

Reply:
done

Reply:
what does `done` mean here? it is still `p` and `super`, right? is that intended?

Reply:
Resolved in #270